transifex/native/django/utils/templates.py

```diff
@@ -60,8 +57,10 @@ def extract_transifex_template_strings(src, origin=None, charset='utf-8'):
     strings = []
     while parser.tokens:
         token = parser.next_token()
-        if token.token_type == TOKEN_BLOCK:
-            source_string, _ = parse_translatable_tag(parser, token)
+        if (token.token_type == TOKEN_BLOCK and
+                token.split_contents()[0] in ('t', 'ut')):
+            tnode = do_t(parser, token)
+            source_string = tnode._to_source_string()
```

Comment:
I have a small concern that the new implementation creates an intermediate `TNode` instance in order to get a `SourceString`, which might affect performance a tiny bit. The change is probably not minimal, so I'm OK with this staying as is.